docs/src/index.js

```diff
@@ -9,6 +9,8 @@ import ReactPerf from 'react-addons-perf';
 import { render } from 'react-dom';
 import App from 'docs/src/components/App';
 
+require('core-js');
```

Comment:
@petermikitsh I guess that replacing `Array.from` is a small enough change to be backed into that PR if you wish :).